docs/docs/guides/web3_migration_guide.md

```diff
@@ -63,3 +63,21 @@ batch.execute();
 // in 4.x
 const responses = await batch.execute();
 ```
+
+### Web3-eth-accounts
+
+#### create and wallet.create
+
+In 1.x `entropy` is an optional parameter for the create method, 4.x does not have `entropy` as a parameter, create method uses an audited package ethereum-cryptography/secp256k1 that is cryptographically secure random number with certain characteristics.
+
+#### privateKeyToAccount
+
+In 1.x `ignoreLength` is an optional parameter for privateKeyToAccount, in 4.x `ignoreLength` is removed
+
+#### signTransaction
+
+In 4.x a transaction object must be passed. Must be typed legacy, EIP2930 or EIP 1559 transaction type Read more about [transaction types] (https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/tx#transaction-types)
+
+#### recover
+
+In 4.x, instead of passing in seperate parameters `messageHash`, `r`, `s`, `v`, a signature object is passed containing those parameters
```

Comment:
If people are using this method, will be a breaking change for them. And the fix seems very easy, so why shouldn't we fix it?